### ISSUE TITLE
Languages other than English

### ISSUE THREAD

**Issue by mxdvl** [guardian/dotcom-rendering#7192]
CAPI supports languages other than english, e.g. [Spanish](http://content.guardianapis.com/search?api-key=test&lang=es) and [Japanese](http://content.guardianapis.com/search?api-key=test&lang=ja)

Screen readers should be informed that part of the page’s content is in a different language, using the [`lang` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang)

**Comment by AshCorr** [comment]
Wow thats interesting! I'm curious as to why we did this, and how speakers of said languages were able to find these articles!
